apps/server/src/modules/files-storage/helper/file-name.spec.ts

```diff
@@ -0,0 +1,334 @@
+import { MikroORM } from '@mikro-orm/core';
+import { ConflictException } from '@nestjs/common';
+import { EntityId } from '@shared/domain';
+import { fileRecordFactory, setupEntities } from '@shared/testing';
+import { ObjectId } from 'bson';
+import { checkDuplicatedNames, hasDuplicateName, modifyFileNameInScope, resolveFileNameDuplicates } from '.';
+import { ErrorType } from '../error';
+
+describe('File Name Helper', () => {
```

Comment:
maybe describe('helper: file-name', () => but let speak in next ticket.

Reply:
Taken to BC-2553